modules/acquia_cms_image/composer.json

```diff
@@ -1,5 +1,10 @@
 {
     "name": "drupal/acquia_cms_image",
     "type": "drupal-module",
-    "description": "Provides an Image media type and related configuration."
+    "description": "Provides an Image media type and related configuration.",
+    "require": {
+        "drupal/acquia_cms_common": "^1",
+        "drupal/field_group": "^3",
+        "drupal/focal_point": "1.5"
```

Comment:
This module also depends on default_content.

Reply:
fixed